Assets/Scripts/Layers/Properties/NewProperties/TransformLayerPropertyData.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Netherlands3D.Coordinates;
+using Newtonsoft.Json;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Netherlands3D.Twin.Layers.Properties
+{
+    [Serializable]
+    public class TransformLayerPropertyData : LayerPropertyData
+    {
+        [SerializeField, JsonProperty] private Coordinate position;
+        [SerializeField, JsonProperty] private Vector3 eulerRotation;
+        [SerializeField, JsonProperty] private Vector3 localScale;
+
+        [JsonIgnore]
+        public Coordinate Position
+        {
+            get => position;
+            set
+            {
+                position = value.Convert(CoordinateSystem.RDNAP);
+                OnPositionChanged.Invoke(position);
+            }
+        }
+
+        [JsonIgnore]
+        public Vector3 EulerRotation
+        {
+            get => eulerRotation;
+            set
+            {
+                eulerRotation = value;
+                OnRotationChanged.Invoke(value);
+            }
+        }
+
+        [JsonIgnore]
+        public Vector3 LocalScale
+        {
+            get => localScale;
+            set
+            {
+                localScale = value;
+                OnScaleChanged.Invoke(value);
+            }
+        }
+
+        [JsonIgnore] public readonly UnityEvent<Coordinate> OnPositionChanged = new();
+        [JsonIgnore] public readonly UnityEvent<Vector3> OnRotationChanged = new();
+        [JsonIgnore] public readonly UnityEvent<Vector3> OnScaleChanged = new();
+        
+        [JsonConstructor]
+        public TransformLayerPropertyData(Coordinate position, Vector3 eulerRotation, Vector3 localScale)
+        {
+            this.position = position.Convert(CoordinateSystem.RDNAP);
```

Comment:
Dit moet niet RDNAP zijn maar de CRS die als referentie is opgegeven in CoordinateSystems; anders krijgen we een gemengde CRS'en in de project data en eigenlijk willen we naar WGS84 ECEF omdat met RD we niet -bijvoorbeeld- de antillen kunnen uitdrukken